dali/sasha/saxref.cpp

```diff
@@ -1515,21 +1591,22 @@ class CNewXRefManager: public CNewXRefManagerBase
         d->getName(scope);
         listDirectory(d,basedir.str(),abort);
         unsigned i = 0;
-        cFileDesc *file = d->files.first(i);
-        while (file) {
-            listOrphans(file,basedir,scope,abort,recentCutoffDays);
-            if (abort)
-                return;
-            file = d->files.next(i);
-        }
-        i = 0;
         cDirDesc *dir = d->dirs.first(i);
         while (dir) {
+            normalizeFileDesc(d,dir,basedir,&mem);
```

Comment:
I think this is too late/should be done earlier in scanLogicalFiles.
Let's discuss offline